.prettierignore

```diff
@@ -1 +1,2 @@
 *.param.yaml
+*.rviz
```

Comment:
Is this the change you are intending to make?

Reply:
It's synced from here.
https://github.com/autowarefoundation/autoware/blob/7c4c396fce51ee2a0c50e3575c77e870e22bebe6/.prettierignore